ScoutSuite/providers/gcp/facade/facade.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
[Same as here](https://github.com/nccgroup/ScoutSuite/pull/229#discussion_r263374292), would remove this line. I won't point out every occurrence but applies to all the other files.